GameData/ROEngines/PartConfigs/RL10A3_SSTU.cfg

```diff
@@ -121,14 +121,8 @@ PART
 	@MODULE[ModuleEngineConfigs]
 	{
 		@configuration = RL10A-1
-		!CONFIG[RL10A-4] {}
-		!CONFIG[RL10A-5] {}
-		!CONFIG[RL10B-2] {}
-		!CONFIG[RL10A-4-1-2] {}
-		!CONFIG[RL10A-4-2N] {}
-		!CONFIG[RL10C-1] {}
-		!CONFIG[RL10C-1-1] {}
-		!CONFIG[RL10C-2-1] {}
-		!CONFIG[RL10C-3] {}
+		!CONFIG:HAS[~name[RL10A-1]] {}
+		!CONFIG:HAS[~name[RL10A-3*]] {}
+		!CONFIG:HAS[~name[CECE*]] {}
```

Comment:
Due to the way MM and HAS works, I don't think this works.  This should delete all configs that don't have the name RL10A-1, then delete all the configs that don't have the name RL10A-3*... so this -should- delete -all- configs.  I think you need a single patch that is the concatenation of the name whitelist:  `!CONFIG:HAS[~name[RL10A-1],~name[RL10A-3*],~name[CECE*]] {}`